Makefile

```diff
@@ -0,0 +1,13 @@
+## builds chain core with c dependencies, for development environments
+build-dev: build-lib
```

Comment:
Do you want `make` (with no args) to invoke this rule? If not, add a default rule before it, something like:
```
null:
    @echo Please specify a make target.
```